### ISSUE TITLE
Translation for the cardHolderNameRequired not working as expected

### ISSUE THREAD

**Issue by placideirandora** [halilb/rn-credit-card#7]
I would like to thank you for working on this productive library!

I have faced an issue while using it though that I would like you to look into and give it a fix:

**CURRENT BEHAVIOR**
I have the following translation object passed to the credit card form component:
```
<CreditCardForm
          LottieView={LottieView}
          horizontalStart
          overrides={{
            labelText: {
              marginTop: 16
            }
          }}
          translations={{
            cardNumber: 'Numéro de Carte',
            cardHolderName: 'Titulaire de Carte',
            nameSurname: 'Nom et Prénom',
            mmYY: 'MM/AA',
            expiration: 'Expiration',
            securityCode: 'Code de Sécurité',
            next: 'Suivant',
            done: 'Terminer',
            cardNumberRequired: 'Numéro de Carte est obligatoire.',
            cardNumberInvalid: 'Ce numéro de carte est invalide.',
            cardHolderNameRequired: 'Nom de titulaire de la carte est obligatoire.',
            cardHolderNameInvalid: 'Ce nom de titulaire de la carte est invalide.',
            expirationRequired: "Date d'Expiration est obligatoire",
            expirationInvalid: "Cette date d'expiration est invalide.",


            securityCodeRequired: 'Code de Sécurité est obligatoire.',
            securityCodeInvalid: 'Ce code de Sécurité est invalide.'
          }}
        />
```
Every field gets translated as expected except the **cardHolderNameRequired** which displays the **cardNumberRequired** instead of its own value.

So, I attempt to skip the card number, they tell me that the card number is required. And I provide it, but when I reach the cardholder's name and attempt the skip, they tell me that the cardholder number is required instead of telling me that the cardholder's name is required. That affects the user experience because it confuses the user.

**EXPECTED BEHAVIOR**
The right corresponding validation message of the cardholder's name should be displayed instead of the incorrect one.

**Comment by halilb** [comment]
Thanks for logging the issue @placideirandora! It appears I've made a mistake on this line: https://github.com/halilb/rn-credit-card/blob/9528e26df3cda48a23a7f45a1dbcad2e7b7e652b/src/components/CreditCardForm.tsx#L150

This looks like a great first issue. Would you like to fix that and open a PR? I'd gladly merge that. Or I can make the change and test it sometime later this week.

**Comment by placideirandora** [comment]
Thanks for recognizing the issue. So, I can make a fix and raise a PR since a minor change and I have time! Let me know if there a documentation to refer to before any contribution. @halilb

**Comment by halilb** [comment]
> Thanks for recognizing the issue. So, I can make a fix and raise a PR since a minor change and I have time! Let me know if there a documentation to refer to before any contribution. @halilb

I'm afraid there is no contribution guideline but this should be pretty straightforward as it's a small change. Please let me know if anything goes wrong.

**Comment by placideirandora** [comment]
Sure. This will be straightforward. I will keep you posted.

**Comment by placideirandora** [comment]
> > Thanks for recognizing the issue. So, I can make a fix and raise a PR since a minor change and I have time! Let me know if there a documentation to refer to before any contribution. @halilb
> 
> I'm afraid there is no contribution guideline but this should be pretty straightforward as it's a small change. Please let me know if anything goes wrong.

I have raised a PR that fixes the issue. You will review it and give it a merge and any feedback for what's not right. [PR Fix](https://github.com/halilb/rn-credit-card/pull/8)

**Comment by halilb** [comment]
Just merged it and published version `1.0.2`. Thanks again @placideirandora!
